quickstartKotlin/src/main/java/com/twilio/video/quickstart/kotlin/VideoActivity.kt

```diff
@@ -0,0 +1,669 @@
+             */
+            localVideoTrack?.let {
+                val enable = !it.isEnabled
+                it.enable(enable)
+                val icon: Int
+                if (enable) {
+                    icon = R.drawable.ic_videocam_white_24dp
+                    switchCameraActionFab.show()
+                } else {
+                    icon = R.drawable.ic_videocam_off_black_24dp
+                    switchCameraActionFab.hide()
+                }
+                localVideoActionFab.setImageDrawable(
+                        ContextCompat.getDrawable(this@VideoActivity, icon))
+            }
+        }
+    }
+
+    private fun muteClickListener(): View.OnClickListener {
+        return View.OnClickListener {
+            /*
+             * Enable/disable the local audio track. The results of this operation are
+             * signaled to other Participants in the same Room. When an audio track is
+             * disabled, the audio is muted.
+             */
+            localAudioTrack?.let {
+                val enable = !it.isEnabled
+                it.enable(enable)
+                val icon = if (enable)
+                    R.drawable.ic_mic_white_24dp
+                else
+                    R.drawable.ic_mic_off_black_24dp
+                muteActionFab.setImageDrawable(ContextCompat.getDrawable(
+                        this@VideoActivity, icon))
+            }
+        }
+    }
+
+    private fun retrieveAccessTokenfromServer() {
+        Ion.with(this)
+                .load("$ACCESS_TOKEN_SERVER?identity=${UUID.randomUUID()}")
+                .asString()
+                .setCallback({ e, token ->
+                    if (e == null) {
+                        this@VideoActivity.accessToken = token
+                    } else {
+                        Toast.makeText(this@VideoActivity,
+                                R.string.error_retrieving_access_token, Toast.LENGTH_LONG)
+                                .show()
+                    }
+                })
+    }
+
+    private fun configureAudio(enable: Boolean) {
+        if (enable) {
+            previousAudioMode = audioManager.mode
+            // Request audio focus before making any device switch
+            requestAudioFocus()
+            /*
+             * Use MODE_IN_COMMUNICATION as the default audio mode. It is required
+             * to be in this mode when playout and/or recording starts for the best
+             * possible VoIP performance. Some devices have difficulties with
+             * speaker mode if this is not set.
+             */
+            audioManager.mode = AudioManager.MODE_IN_COMMUNICATION
+            /*
+             * Always disable microphone mute during a WebRTC call.
+             */
+            previousMicrophoneMute = audioManager.isMicrophoneMute
+            audioManager.isMicrophoneMute = false
+        } else {
+            audioManager.mode = previousAudioMode
```

Comment:
Multiple method invocations can be called using function [with](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/with.html).